scripts/contract.mjs

```diff
@@ -53,11 +53,11 @@ export async function verify(address, contract, args = undefined)  {
       contract,
       process.env.ETHERSCAN_API_KEY,
       "--chain",
-      process.env.CHAIN,
+      process.env.CHAIN_ID,
```

Comment:
why the switch?

Reply:
`CHAIN` env variable seems to be used in the forge command line. This let's us leverage the ENV variable w/o overriding Forge.

Would love to pair with you on this tomorrow to see if I'm missing something. 🔍